saleor/settings.py

```diff
@@ -128,6 +128,8 @@ def get_bool_from_env(name, default_value):
 EMAIL_USE_TLS = email_config["EMAIL_USE_TLS"]
 EMAIL_USE_SSL = email_config["EMAIL_USE_SSL"]
 
+ENABLE_ACCOUNT_CONFIRMATION_BY_EMAIL = True
```

Comment:
It should take the value from env and shouldn't be true by default, otherwise it could be a breaking or unwanted change

Reply:
cc @maarcingebala that could be up for discussion whether it should be `True` or `False` out of the box. Or maybe later on.

Reply:
@NyanKiyoshi Discussed with Marcin, we think it should be enabled by default as it's the default for most of the internet.